keras_rs/src/types.py

```diff
@@ -0,0 +1,25 @@
+"""Type definitions."""
+
+from typing import Any, Optional, Sequence
+
+"""
+A tensor in any of the backends.
+
+We do not define it explicitly to not require all the backends to be installed
+and imported. The explicit definition would be:
+```
+Union[
```

Comment:
Might also include NumPy array (for numpy backend + fact that layers will accept and convert any numpy array)

Reply:
Added.